client/package.json

```diff
@@ -12,6 +14,7 @@
     "@types/react": "^18.0.21",
     "@types/react-dom": "^18.0.6",
     "dayjs": "^1.11.6",
+    "ethers": "^5.7.2",
```

Comment:
`ethers` is using [BN.js](https://www.npmjs.com/package/bn.js) under the hood, maybe we can use it directly since we don't do any Ethereum-related stuff